lang/user/id.xml

```diff
@@ -223,19 +219,18 @@ Kode kesalahan: {}</an_error_occurred>
 			<days>31</days>
 		</small_mday>
 		<hidden_trophy>Trofi Tersembunyi</hidden_trophy>
-		<one_hour_ago>1 Jam Yg Lalu</one_hour_ago>
-		<one_minute_ago>1 Minit Yg Lalu</one_minute_ago>
+		<one_hour_ago>1 Jam Yang Lalu</one_hour_ago>
+		<one_minute_ago>1 Menit Yang Lalu</one_minute_ago>
 		<bronze>Perunggu</bronze>
 		<gold>Emas</gold>
-		<platinum>Platinum</platinum>
 		<silver>Perak</silver>
-		<hours_ago>{} Jam Yg Lalu</hours_ago>
-		<minutes_ago>{} Minit Yg Lalu</minutes_ago>
+		<hours_ago>{} Jam Yang Lalu</hours_ago>
+		<minutes_ago>{} Menit Yang Lalu</minutes_ago>
 	</common>
 
 	<compatibility name="Compatibility">
 		<states>
-			<state id = "-1">Ga diketahui</state>
+			<state id = "-1">Tidak Diketahui</state>
```

Comment:
> @darkash here.

We've decided to accept this part since at the time of the initial review, I didn't know the reasoning for the change. We did exchange our thoughts about this in DM the other day. Sorry for some trouble :)

Reply:
> > @darkash here.
> 
> We've decided to accept this part since at the time of the initial review, I didn't know the reasoning for the change. We did exchange our thoughts about this in DM the other day. Sorry for some trouble :)

okay, i push it again